python/ray/tests/test_actor_failures.py

```diff
@@ -630,6 +653,8 @@ def test_checkpointing_save_exception(ray_start_regular,
                                       ray_checkpointable_actor_cls):
     """Test actor can still be recovered if checkpoints fail to complete."""
 
+    p = init_pubsub()
```

Comment:
Can we move it to fixture instead of repeating?

Reply:
Maybe we can also add unsubscribe in the fixture.

Reply:
@rkooo567  I think we still need to repeat it even though we move it to fixture, because `p` is used in `get_error_message` fuction. I don't know much about Python, is there other way that we can avoid repeating it ?

Reply:
Move it to fixture doesn't work, the subscibe must happed after `ray.init`, because the `init_pubsub` uses `ray.worker.global_worker.redis_client` (it is `None` if we move `init_pubsub` to fixture)